docs/configs.md

```diff
@@ -10,7 +10,7 @@ The following is the list of options that `rapids-plugin-4-spark` supports.
 On startup use: `--conf [conf key]=[conf value]`. For example:
 
 ```
-${SPARK_HOME}/bin/spark --jars 'rapids-4-spark_2.12-0.6.0-SNAPSHOT.jar,cudf-21.06-SNAPSHOT-cuda11.jar' \
+${SPARK_HOME}/bin/spark --jars 'rapids-4-spark_2.12-21.06.0-SNAPSHOT.jar,cudf-21.06-SNAPSHOT-cuda11.jar' \
```

Comment:
Interesting that we're going with 21.06.0-SNAPSHOT in the plugin but 21.06-SNAPSHOT in cudf?  Should cudf be using 21.06.0-SNAPSHOT as well?

Reply:
yes, I am waiting for the final call for 21.06.0 vs 21.06.00 to make the change for both sides.

we used to not add the patch number for .0 release which was not good~